src/routes/Login.svelte

```diff
@@ -13,7 +13,8 @@
     }
 </script>
 
-Login page
-
-<Button label={I18n.ts("landing.loginStudent")} className="short" href={I18n.ts("landing.loginStudent")} onClick={() => logIn(role.STUDENT)}/>
-<Button label={I18n.ts("landing.loginTeacher")} className="short" href={I18n.ts("landing.loginTeacher")} onClick={() => logIn(role.TEACHER)}/>
+<div>
+    Login page
+    <Button label={I18n.ts("landing.loginStudent")} className="short" href={I18n.ts("landing.loginStudent")} onClick={() => logIn(role.STUDENT)}/>
+    <Button label={I18n.ts("landing.loginTeacher")} className="short" href={I18n.ts("landing.loginTeacher")} onClick={() => logIn(role.TEACHER)}/>
```

Comment:
This class is unused. Also, the `href` value looks a bit odd 

![image](https://user-images.githubusercontent.com/22448633/74229714-3503da00-4cc3-11ea-825a-72950751ed10.png)